Sources/BraveWallet/Crypto/Transaction Confirmations/TransactionStatusView.swift

```diff
@@ -0,0 +1,181 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+import BigNumber
+import Strings
+import DesignSystem
+
+struct TransactionStatusView: View {
+  @ObservedObject var confirmationStore: TransactionConfirmationStore
+  let networkStore: NetworkStore
+  @Binding var transactionDetails: TransactionDetailsStore?
+ 
+  let onDismiss: () -> Void
+  
+  @Environment(\.openWalletURLAction) private var openWalletURL
+  
+  @ViewBuilder private var loadingTxView: some View {
+    VStack {
+      TxProgressView()
+    }
```

Comment:
Doesn't need to be in a `VStack`

Reply:
Using `ProgressView` with `.braveCircular` style. Move this entirely out of `TransactionStatusView` at [02e39dd](https://github.com/brave/brave-ios/pull/6960/commits/02e39dd3500416bf27c899accce6252b59e78d3f)